pkg/kube/controllers/extendedstatefulset/pod_webhook.go

```diff
@@ -34,7 +34,7 @@ func NewExtendedStatefulsetPodMutator(log *zap.SugaredLogger, config *config.Con
 			},
 		},
 		Path: "/mutate-pods",
-		Name: "mutate-pods.fissile.cloudfoundry.org",
+		Name: "mutate-pods.quarks.cloudfoundry.org",
```

Comment:
any reason why you are not using the https://github.com/cloudfoundry-incubator/quarks-utils/blob/af677c684e378cce33c908f8ef640ed39b60e78f/pkg/names/apigroup.go#L4 in here?